sdk/security/keyvault/azcertificates/README.md

```diff
@@ -56,7 +57,7 @@ func main() {
 
 ### Client
 
-With a [Client][client_docs] you can get certificates from the vault, create new certificates and
+With a azcertificates client, you can get certificates from the vault, create new certificates and
```

Comment:
`Client` is the type though, and the only one of that name in the module

Reply:
fixed in [d93dca3](https://github.com/Azure/azure-sdk-for-go/pull/20681/commits/d93dca3f6b14b8b4f1d40b620c1c6f7002f5662b)